DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/activity/activity.service.js

```diff
@@ -0,0 +1,40 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.activity')
+        .factory('ActivityService', activityService);
+
+    activityService.$inject = ['$q', '$http', 'ACTIVITY_CONSTANTS'];
+
+    function activityService($q, $http, constants) {
+        var defer;
```

Comment:
Hm, I thought about it this way, but then decided that it is too simple. Over Engineered again(